### DISCUSSION TITLE
Command line argument to run X68000 games with midi support?

### DISCUSSION THREAD

**Discussion by HazukiXX** [mamedev/discussions#122] [Q&A] [answered]
For example, let's say I want to play Akumajou Dracula with midi support. You'd need to trigger that sound mode selection screen at the beginning of the game, which lets you choose between different sound sources. This is the screen in question:
![screen](https://github.com/user-attachments/assets/811304be-6f56-4e87-8edc-f1b651025c10)
If I do only this:
`mame x68000 akumajo`
The game bypasses that screen entirely and you only get the regular FM sound source, which makes me think extra arguments are required to trigger it. If that's the case, can someone point me to what I need to write there exactly?

**Comment by einstein95** [comment, marked as answer]
`mame x68000 akumajo -exp1 x68k_midi`
![0000](https://github.com/user-attachments/assets/5a77dd24-d8eb-486b-a4b5-87f8605a0cda)

**Reply by HazukiXX** [reply]
Thank you!
